content/200-concepts/100-components/02-prisma-client/080-null-and-undefined.mdx

```diff
@@ -134,3 +134,90 @@ updateUser: (parent, args, ctx: Context) => {
   })
 },
 ```
+
+## The effect of <inlinecode>null</inlinecode> and <inlinecode>undefined</inlinecode> on conditionals
+
+There are some caveats to filtering with conditionals which might produce unexpected results. When filtering with conditionals you might expect one result but receive another given how Prisma treats nullable values.
+
+### A high level overview
```

Comment:
I think we can remove this heading and just introduce the table instead - e.g. "The following table provides a high-level overview of how different operators handle 0, 1, and `n` filters:"